test/Interpreter/SDK/objc_getClass.swift

```diff
@@ -195,6 +220,22 @@ testSuite.test("ResilientField")
   expectEqual(ResilientFieldSubclassObjC().subvalue.i, 4)
 }
 
+testSuite.test("ResilientNSObject")
+  .skip(.custom({ getClassHookMissing },
```

Comment:
Any reason why this one isn't `requireOwnProcess`?

Reply:
Oh, how did I miss that?